src/enzo/Grid_ComputeTimeStep.C

```diff
@@ -443,6 +448,46 @@ float grid::ComputeTimeStep()
     delete [] cooling_time;
   }
 
+   /*FDM:  Calculate minimum dt due to quantum pressure. */
+
+  if(QuantumPressure){
+  float TemperatureUnits = 1.0, DensityUnits = 1.0, LengthUnits = 1.0;
+  float VelocityUnits = 1.0, TimeUnits = 1.0, aUnits = 1.0;
+  double MassUnits = 1.0;
+  float hmcoef=1.0;
+
+
+      if (QuantumGetUnits(&DensityUnits, &LengthUnits, &TemperatureUnits, &TimeUnits, &VelocityUnits, &MassUnits, Time) == FAIL) {
+    ENZO_FAIL("Error in GetUnits.");
+       }
+
+
+      hmcoef = 5.9157166856e27*TimeUnits/pow(LengthUnits,2)/FDMMass;
```

Comment:
For the sake of documentation can you specify what `hmcoef` is and what the hard-coded number is ( or maybe its obvious and I'm just not familiar with the physics )

Reply:
fixed